contracts/market/MarketsRegistry.sol

```diff
@@ -17,6 +18,16 @@ import "../amm/InitializeableAmm.sol";
 contract MarketsRegistry is OwnableUpgradeSafe, Proxiable, IMarketsRegistry {
     /** Use safe ERC20 functions for any token transfers since people don't follow the ERC20 standard */
     using SafeERC20 for IERC20;
+    using SafeMath for uint256;
+
+    struct Receiver {
+        address secondaryAddress;
+        uint8 vaultPercentage;
```

Comment:
removed secondary address percentage, so the remainder after vault percentage calculation is the amount to be sent to the secondary address